test/e2e/kind_test.go

```diff
@@ -367,6 +373,48 @@ func (s *kindSuite) TestKindRun() {
 
 	})
 
+	s.T().Run("APM (hostPort, local service, UDP) works", func(t *testing.T) {
```

Comment:
```suggestion
	s.T().Run("APM hostPort, local service, UDP works", func(t *testing.T) {
```
Could we remove parentheses in the test names? That way when you run a specific test using `make e2e-test` we don't have to add double quotes or escape characters to account for the parentheses